Build/INSTALL.md

```diff
@@ -9,16 +9,15 @@ Then, follow the guide based on your operating system.
 
 Windows
 ===
-Using Visual Studio, simply build and it will place the .exe file in the correct directory.
+Using Visual Studio, simply build and it will place the .exe file in the correct directory. You will need Visual Studio, MSVC, C++ ATL, C++ MFC, and the Windows SDK; all of which can be installed with Visual Studio Installer.
```

Comment:
I think making this into a bulleted list is easier. And even better is a screenshot from the installer since it'd be more visual.

Reply:
> I think making this into a bulleted list is easier. And even better is a screenshot from the installer since it'd be more visual.

Done in the newest commit.